app/controllers/AssessmentController.scala

```diff
@@ -63,24 +63,20 @@ class AssessmentController @Inject() (
             .get(CategorisationDetailsQuery(recordId))
             .flatMap { categorisationInfo =>
               categorisationInfo.getAssessmentFromIndex(index).map { assessment =>
-                val listItems = assessment.getExemptionListItems
```

Comment:
Not sure this is part of your ticket but the CYA stuff on this is broken so when you try to change something from the change link on the CYA page it doesn't persist

Reply:
There's a linked ticket for CYA in the blocked section :)